doc/content/gettingstarted/mqtt/subscribe.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Subscribing to messages"
+description: ""
+weight: 1
+---
+
+The Application Server publishes on the following topics:
+
+- `v3/{application id}/devices/{device id}/join`
+- `v3/{application id}/devices/{device id}/up`
+- `v3/{application id}/devices/{device id}/down/queued`
+- `v3/{application id}/devices/{device id}/down/sent`
+- `v3/{application id}/devices/{device id}/down/ack`
+- `v3/{application id}/devices/{device id}/down/nack`
+- `v3/{application id}/devices/{device id}/down/failed`
+
+While you could subscribe to separate topics, for the tutorial subscribe to `#` to subscribe to all messages.
+
+With your MQTT client subscribed, when a device joins the network, a `join` message gets published. For example, for a device ID `dev1`, the message will be published on the topic `v3/app1/devices/dev1/join` with the following contents:
+
+```json
+{
+  "end_device_ids": {
+    "device_id": "dev1",
+    "application_ids": {
+      "application_id": "app1"
+    },
+    "dev_eui": "4200000000000000",
+    "join_eui": "4200000000000000",
+    "dev_addr": "01DA1F15"
+  },
+  "correlation_ids": [
+    "gs:conn:01D2CSNX7FJVKQPCVG612QF1TX",
+    "gs:uplink:01D2CT834K2YD17ZWZ6357HC0Z",
+    "ns:uplink:01D2CT834KNYD7BT2NHK5R1WVA",
+    "rpc:/ttn.lorawan.v3.GsNs/HandleUplink:01D2CT834KJ4AVSD1SJ637NAV6",
+    "as:up:01D2CT83AXQFQYQ35SR74CTWKH"
```

Comment:
`as:up:01` renders a funny character in my browser:
```"as🆙01D2CV8HNGJ57G25BW0FCZNY07"```

Reply:
Fixed